build-data-capturing-gradle-samples/capture-test-pts-support/gradle-test-pts-support.gradle.kts

```diff
@@ -56,6 +57,19 @@ class Capture(val logger: Logger) {
         try {
             var engines = t.classpath.files.stream().filter { f -> f.name.endsWith(".jar") }
                 .filter { f -> supportedEngines.values.stream().anyMatch { e -> f.name.contains(e) } }
+                .filter { f ->
+                    if (f.name.contains("kotest-runner")) {
+                        val kotestVersionString = f.name.split("-")[f.name.split("-").lastIndex].replace(".jar", "")
+                        val kotestVersion = VersionNumber.parse(kotestVersionString)
+                        if (VersionNumber.UNKNOWN == kotestVersion) {
+                            logger.warn("Unable to parse kotest version from file name ${f.name}")
+                            return@filter false
+                        }
+                        VersionNumber.parse("5.6.0") <= kotestVersion
```

Comment:
According to the JavaDoc, this [VersionNumber](https://docs.gradle.org/current/javadoc/org/gradle/util/VersionNumber.html) will be removed in Gradle 9.0. 

There is an internal one [`org.gradle.util.internal.VersionNumber`](https://github.com/gradle/gradle/blob/e64caf03d0ceed251f83a3e90281a4d8c5dc5f7e/platforms/core-runtime/base-services/src/main/java/org/gradle/util/internal/VersionNumber.java#L36). Should we use that instead?

Reply:
Fixed

Reply:
And yeah I guess to answer the question, I think it makes sense to use that one, since the external one will be removed soon-ish, though I'm not sure if we should completely refrain from using internal packages